project.clj

```diff
@@ -93,8 +95,6 @@
                                 ;; Compress the output?
                                 :pretty-print? false}}]}
 
-  :prep-tasks [["garden" "once"]]
```

Comment:
@Pireax 
Breaks production build as it doesn't create the CSS
![image](https://user-images.githubusercontent.com/3913859/90669280-9d052100-e20e-11ea-864e-cf24b9194f65.png)

Please put this back in.